src/components/atoms/KeywordReview.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+
+const KeywordReview = ({ keyword, reviewCount }) => {
+  const [fillPercentage, setFillPercentage] = useState(0);
```

Comment:
데이터를 단순하게 가공만 한다면 상태가 아닙니다.

```suggestion
  const fillPercentage = (reviewCount / 200) * 100;
```